ivy/stateful/module.py

```diff
@@ -516,7 +520,7 @@ def __call__(
 
     def __getattribute__(self, name):
         if name == "v":
-            if super().__getattribute__("v") is None and not self.built:
```

Comment:
This check has been there for a while (like a year or so), any reasons for removing it (just out of curiosity)?

Reply:
Yep sounds good, in which case I think we can probably add back `if not super().__getattribute__("_v") and not self.built:`, given that an empty container is a boolean `False`. What do you think?